crates/biome_markdown_parser/tests/md_test_suite/err/invalid_header.md.snap

```diff
@@ -0,0 +1,207 @@
+        46: MD_TEXTUAL@54..55
+          0: MD_TEXTUAL_LITERAL@54..55 "r" [] []
+        47: MD_TEXTUAL@55..56
+          0: MD_TEXTUAL_LITERAL@55..56 "s" [] []
+        48: MD_TEXTUAL@56..58
+          0: MD_TEXTUAL_LITERAL@56..58 "#" [Newline("\n")] []
+        49: MD_TEXTUAL@58..59
+          0: MD_TEXTUAL_LITERAL@58..59 "#" [] []
+        50: MD_TEXTUAL@59..60
+          0: MD_TEXTUAL_LITERAL@59..60 "S" [] []
+        51: MD_TEXTUAL@60..61
+          0: MD_TEXTUAL_LITERAL@60..61 "t" [] []
+        52: MD_TEXTUAL@61..62
+          0: MD_TEXTUAL_LITERAL@61..62 "i" [] []
+        53: MD_TEXTUAL@62..63
+          0: MD_TEXTUAL_LITERAL@62..63 "l" [] []
+        54: MD_TEXTUAL@63..65
+          0: MD_TEXTUAL_LITERAL@63..65 "l" [] [Whitespace(" ")]
+        55: MD_TEXTUAL@65..66
+          0: MD_TEXTUAL_LITERAL@65..66 "I" [] []
+        56: MD_TEXTUAL@66..67
+          0: MD_TEXTUAL_LITERAL@66..67 "n" [] []
+        57: MD_TEXTUAL@67..68
+          0: MD_TEXTUAL_LITERAL@67..68 "v" [] []
+        58: MD_TEXTUAL@68..69
+          0: MD_TEXTUAL_LITERAL@68..69 "a" [] []
+        59: MD_TEXTUAL@69..70
+          0: MD_TEXTUAL_LITERAL@69..70 "l" [] []
+        60: MD_TEXTUAL@70..71
+          0: MD_TEXTUAL_LITERAL@70..71 "i" [] []
+        61: MD_TEXTUAL@71..72
+          0: MD_TEXTUAL_LITERAL@71..72 "d" [] []
+        62: MD_TEXTUAL@72..74
+          0: MD_TEXTUAL_LITERAL@72..74 "#" [Newline("\n")] []
+        63: MD_TEXTUAL@74..75
+          0: MD_TEXTUAL_LITERAL@74..75 "#" [] []
+        64: MD_TEXTUAL@75..76
+          0: MD_TEXTUAL_LITERAL@75..76 "#" [] []
+        65: MD_TEXTUAL@76..77
+          0: MD_TEXTUAL_LITERAL@76..77 "N" [] []
+        66: MD_TEXTUAL@77..78
+          0: MD_TEXTUAL_LITERAL@77..78 "o" [] []
+        67: MD_TEXTUAL@78..80
+          0: MD_TEXTUAL_LITERAL@78..80 "t" [] [Whitespace(" ")]
+        68: MD_TEXTUAL@80..81
+          0: MD_TEXTUAL_LITERAL@80..81 "V" [] []
+        69: MD_TEXTUAL@81..82
+          0: MD_TEXTUAL_LITERAL@81..82 "a" [] []
+        70: MD_TEXTUAL@82..83
+          0: MD_TEXTUAL_LITERAL@82..83 "l" [] []
+        71: MD_TEXTUAL@83..84
+          0: MD_TEXTUAL_LITERAL@83..84 "i" [] []
+        72: MD_TEXTUAL@84..86
+          0: MD_TEXTUAL_LITERAL@84..86 "d" [] [Whitespace(" ")]
+        73: MD_TEXTUAL@86..87
+          0: MD_TEXTUAL_LITERAL@86..87 "E" [] []
+        74: MD_TEXTUAL@87..88
+          0: MD_TEXTUAL_LITERAL@87..88 "i" [] []
+        75: MD_TEXTUAL@88..89
+          0: MD_TEXTUAL_LITERAL@88..89 "t" [] []
+        76: MD_TEXTUAL@89..90
+          0: MD_TEXTUAL_LITERAL@89..90 "h" [] []
+        77: MD_TEXTUAL@90..91
+          0: MD_TEXTUAL_LITERAL@90..91 "e" [] []
+        78: MD_TEXTUAL@91..93
+          0: MD_TEXTUAL_LITERAL@91..93 "r" [] [Whitespace(" ")]
+  2: EOF@93..93 "" [] []
+
+```
+
+## Diagnostics
+
+```
+invalid_header.md:1:1 parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Invalid header format: missing space after '#'
```

Comment:
Markdown is a markup language, and it's also a very lax language. This means that there aren't any errors in markdown. When something, in this case an header, turns out to NOT be an header, a paragraph is parsed.

Check this [example](https://astexplorer.net/#/gist/18adca50f323a2fffa28d99eb97b4b53/6eaed59ea7b0799215af2116aef9ee4aa9d9fbe0)

Reply:
I'll make the necessary changes and test again to see if they disappear.

Reply:
@ematipico I had the doubt if a `#header` wouldn't be formatted to add the space. But I see that prettier doesn't do that either. That does change some things. I'll try to update this today.

Reply:
Please read the spec, it's very important